packages/@okta/vuepress-site/docs/reference/api/authn/index.md

```diff
@@ -91,9 +91,17 @@ The context object allows [trusted web applications](#trusted-application) such
 | ----------- | ----------------------------------------------------------------------------- | -------- | -------- | ------ | -------- | --------- | --------- |
 | deviceToken | A globally unique ID identifying the user's client device or user agent | String   | TRUE     | FALSE  | FALSE    |           | 32        |
 
-> **Note:** You must always pass the same `deviceToken` for a user's device with every authentication request for **per-device** or **per-session** Sign-On Policy Factor challenges. If the `deviceToken` is absent or does not match the previous `deviceToken`, the user is challenged every-time instead of **per-device** or **per-session**.
+> **Note:** You must always pass the same `deviceToken` for a user's device with every authentication request for **per-device** or **per-session** Sign-On Policy Factor challenges. If the `deviceToken` is absent or does not match the previous `deviceToken`, the user is challenged every-time instead of **per-device** or **per-session**. Similarly you must always pass the same `deviceToken` for user's device with every authentication request for **new device security behavior detection**. If the `deviceToken` is absent or does not match a recent `deviceToken` for the user, the request will be considered to be from a new device. For more information about this feature, see [Security Behavior Detection](https://help.okta.com/en/prod/Content/Topics/Security/proc-security-behavior-detection.htm).
```

Comment:
Kept the existing note and added everything starting at `Similarly`

Reply:
i'd rather not touch the old content since it is unrelated to my team. If we need to clean up the wording can someone do that in a separate PR? I'll add a new paragraph and take your wording for my content

Reply:
I wrote the existing content. :)

Reply:
I can change it later if you'd prefer.

Reply:
oh didn't realize. I feel like you can drop the mic on that one. I'll update it then :)